aries-site/src/data/wcag/components.json

```diff
@@ -12,6 +12,24 @@
       { "rule": "2.4.9", "status": "conditional" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
+      { "rule": "1.4.3", "status": "passed" },
+      { "rule": "2.1.1", "status": "passed" },
+      { "rule": "2.1.2", "status": "passed" },
+      { "rule": "2.4.7", "status": "passed" },
+      { "rule": "2.4.11", "status": "passed" },
+      { "rule": "2.4.12", "status": "passed" },
```

Comment:
I might not include these, it's dependent on how an app is built and if content gets placed over the menu component. It isn't something that is directly related to the menu component itself

Reply:
good point